### PR TITLE
add assign/unassign and reorder presentation speakers feature

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=2889010&fileview=grid

### CODE REVIEW

File: src/components/forms/event-form.js [hunk 1]
```diff
@@ -977,19 +1027,35 @@ render() {
...
-                            getOptionLabel={(speaker) => `${speaker.first_name} ${speaker.last_name} (${speaker.email})`}
-                        />
+                <>
+                    <div className="row form-group">
+                        <div className="col-md-10">
+                            <label> {T.translate("general.speakers")} </label>
+                            <SpeakerInput
+                                id="speaker"
+                                onChange={this.handleChange}
+                                summitId={currentSummit.id}
```
Comment (smarcet) [inline src/components/forms/event-form.js:1037]: this is not correct u should search by all speakers

File: src/components/forms/event-form.js [hunk 2]
```diff
@@ -977,19 +1027,35 @@ render() {
...
-                            history={history}
-                            getOptionLabel={(speaker) => `${speaker.first_name} ${speaker.last_name} (${speaker.email})`}
-                        />
+                <>
+                    <div className="row form-group">
+                        <div className="col-md-10">
+                            <label> {T.translate("general.speakers")} </label>
+                            <SpeakerInput
+                                id="speaker"
+                                onChange={this.handleChange}
```
Comment (smarcet) [inline src/components/forms/event-form.js:1036]: DDL should be declared as clearable

Comment (smarcet) [inline src/components/forms/event-form.js:1036]: on change should reset the current value

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: src/components/forms/event-form.js (hunk 1 above)
Comment (romanetar) [reply src/components/forms/event-form.js]: fixed

File: src/components/forms/event-form.js (hunk 2 above)
Comment (romanetar) [reply src/components/forms/event-form.js:1037]: done

Comment (romanetar) [reply src/components/forms/event-form.js:1037]: fixed

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**